dev/format_code.sh

```diff
@@ -78,8 +81,7 @@ do
   done
 done
 
-echo "Finished: $COUNTER files modified."
-
+echo "Finished: $COUNTER files modified." 
```

Comment:
Hmm what happened here? ;)

Reply:
Huh no idea. I normally look at the git diff before commiting... Fixed.